packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2437,9 +2440,18 @@ export class ContainerRuntime
 					throw error2;
 				}
 			} else {
-				// pre-0.58 error message: orderSequentiallyCallbackException
-				this.closeFn(new GenericError("orderSequentially callback exception", error));
+				this.closeFn(
+					wrapError(
+						error,
+						(errorMessage) =>
+							new GenericError("orderSequentially callback exception", error, {
+								errorMessage,
+								orderSequentiallyCalls: this._orderSequentiallyCalls,
+							}),
```

Comment:
Elsewhere I've espoused the pattern of putting the inner error message after a colon.  It puts it front-and-center when reading logs, but also makes it easy to parse out if you want to group by all `orderSequentially callback exception`s.

```suggestion
							new GenericError(`orderSequentially callback exception: ${errorMessage}`, error, {
								orderSequentiallyCalls: this._orderSequentiallyCalls,
							}),
```